### DISCUSSION TITLE
How do I use this?

### DISCUSSION THREAD

**Discussion by graue** [tvler/streetpass#72] [Q&A] [answered]
I was browsing my addons in Firefox (131.0 on Linux) and remembered I've had StreetPass installed for a while, but it's never done anything.

The readme says, "StreetPass lets you know when you've found one of these [Mastodon verification] links, and adds them to your StreetPass list." But it never lets me know I found one. I don't see anything when I go to, for instance, https://tinysubversions.com, which has a rel="me" link to a Mastodon account. The instructions don't say how to access this StreetPass list.

There's a screenshot that seems to show a StreetPass icon just to the right of the URL bar. But I don't have that icon. Nor do I see adding that icon as an option when I go into Customize Toolbar. So, how do I actually see the list?

**Comment by tvler** [comment, marked as answer]
Thanks for trying out StreetPass Scott! Here's how I was able to pin the streetpass icon to my toolbar in a fresh installation of Firefox on my mac. I believe it's a similar pattern to do this on linux as well.

https://github.com/user-attachments/assets/dfa963d6-c10a-4ef3-b2d7-2bbafeaf3bdc

**Reply by graue** [reply]
Aha. Thank you. Other add-ons have shown up without making me do that, but I guess that's one of the mysteries of the universe. Nice to have it working!
